src/ansys/geometry/core/misc/accuracy.py

```diff
@@ -54,6 +62,23 @@ def length_is_equal(comparison_length: Real, reference_length: Real) -> bool:
             comparison_length, reference_length, LENGTH_ACCURACY, LENGTH_ACCURACY
         )
 
+    def equal_doubles(a: Real, b: Real):
+        """Compare two double values."""
+        return Accuracy.is_within_tolerance(a, b, DOUBLE_ACCURACY, DOUBLE_ACCURACY)
+
+    def compare_with_tolerance(
+        a: Real, b: Real, relative_tolerance: Real, absolute_tolerance: Real
+    ) -> Real:
+        """Compare two doubles given the relative and absolute tolerances."""
+        if Accuracy.is_within_tolerance(a, b, relative_tolerance, absolute_tolerance):
+            return 0
+        if a < b:
+            return -1
+        elif a == b:
+            return 0
+        else:
+            return 1
```

Comment:
```suggestion
        elif a < b:
            return -1
        else:
            return 1
```

Doesn't it make sense to have it like this?